pyads/get_bad_active_creatives.py

```diff
@@ -0,0 +1,177 @@
+                 .WithBindVariable('creativeId', creative_id))
+
+    while True:
+      response = creative_service.getCreativesByStatement(statement.ToStatement())
+      if 'results' in response and len(response['results']):
+        for creative in response['results']:
+          db_creatives_to_scan[str(creative['id'])] = creative
+        statement.offset += statement.limit
+      else:
+        break
+
+  print('Numer of creatives feteched: {}'.format(len(db_creatives_to_scan.items())))
+
+
+def main(client):
+  parser = argparse.ArgumentParser(description='--fetch if you want to fetch dfp configs')
+  parser.add_argument('--fetch', help='use if you want to fetch data', action="store_true")
+  args = parser.parse_args()
+
+  text_to_look_for = "http://"
+
+  if args.fetch:
+    input("Press Enter to FETCH ...")
+    fetch_line_items(client)
+    fetch_associatiated_creatives(client)
+  else:
+    creatives_to_scan = {}
+
+    db_line_items = shelve.open('data/line_items.db')
+    db_creatives = shelve.open('data/creatives.db')
+
+    active_line_items = db_line_items['active_line_items']
+    creatives = db_creatives['creatives_to_scan']
+
+    for creative_id, line_item_id in tqdm(creatives.items()):
+      if line_item_id in active_line_items:
+        creatives_to_scan[creative_id] = creatives[creative_id]
+
+    print('Numer of creatives associatiated with line items: {}'.format(len(creatives_to_scan)))
+
+    if args.fetch:
+      fetch_creatives(client, creatives_to_scan)
+    else:
+      db_bad_creatives = shelve.open('data/bad_creatives.db')
+
+      bad_creatives = {}
+
+      db_creatives_to_scan = shelve.open('data/creatives_to_scan.db')
+
+      print(len(creatives_to_scan))
+
+      for creative_id, creative in tqdm(db_creatives_to_scan.items()):
+        if 'snippet' in creative and text_to_look_for in creative['snippet']:
+          bad_creatives[creative['id']] = creative['name']
+          db_bad_creatives[str(creative['id'])] = creative
+          continue
+        if 'expandedSnippet' in creative and text_to_look_for in creative['expandedSnippet']:
+          bad_creatives[creative['id']] = creative['name']
+          db_bad_creatives[str(creative['id'])] = creative
+          continue
+        if 'htmlSnippet' in creative and text_to_look_for in creative['htmlSnippet']:
+          bad_creatives[creative['id']] = creative['name']
+          db_bad_creatives[str(creative['id'])] = creative
+          continue
+
+    print('Numer of bad creatives found: {}'.format(len(bad_creatives)))
+
+    workbook = xlsxwriter.Workbook('bad_creatives.xlsx')
+    worksheet = workbook.add_worksheet()
+
+    row = 0
+    col = 0
+
+    for creative_id in bad_creatives.keys():
+        worksheet.write(row, col, creative_id)
+        row += 1
+
+
+if __name__ == '__main__':
+  yaml_string = "ad_manager: " + "\n" + \
+                "  application_name: Wikia - DFP\n" + \
+                "  network_code: " + str(5441) + "\n" + \
```

Comment:
what is the advantage of using `str` here instead of `"  network_code: 5441\n"`? Unless you planned moving `5441` to a variable, then please do so :)